cnnlabs/cnn-antools-factors-fe
# CNNETANTWO-21 Live Streams Apple News,  Page and Digital Feed Wrapper Module

This feature is a lives streams module that allows the user to toggle livestreams on or off, associate each stream to a certain url, and also view live stream video.

Fixes: CNNETANTWO-78, CNNETANTWO-90, CNNETANTWO-80
[CNNETANTWO-78](http://tickets.turner.com/browse/CNNETANTWO-78)  Connected Page for the Live Streams page
[CNNETANTWO-90](http://tickets.turner.com/browse/CNNETANTWO-90) Create Digital Feed Wrapper Module
[CNNETANTWO-80](http://tickets.turner.com/browse/CNNETANTWO-80) Create URL input Module

![image](https://user-images.githubusercontent.com/13541552/28927215-00508b6c-7838-11e7-91bc-cf06d4d4901c.png)
![image](https://user-images.githubusercontent.com/13541552/28927234-0bdce444-7838-11e7-82cf-7ef563587f4a.png)
![image](https://user-images.githubusercontent.com/13541552/28927245-136055fc-7838-11e7-8df3-b859a6511675.png)

Reviewers: Jenny Vallon <jenny.vallon@turner.com>, Sean Joseph <sean.joesph@turner.com>, Alex Sover <alex.sover@turner.com>

To test:
Pull this branch down and goto: http://localhost:5050/streams
Click link article to test link article dialogue box. Also try to link article.
Click toggle to test toggle dialogue box.

Known Bugs (Will be added in another polishing ticket for url module):
1. if you input nothing into the link article dialogue and click link, you will get the last value inputed into the module.
2. No url validation on the url module.
3."LINK ARTICLE" does not look like a link and is just plain text. 
4. Link Icon does not appear next to link article.
